src/main/scala/util/RunUtils.scala

```diff
@@ -539,13 +559,16 @@ object RunUtils {
     val analysisResult = mutable.ArrayBuffer[StaticAnalysisContext]()
     while (modified) {
       Logger.debug("[!] Running Static Analysis")
-      val result = StaticAnalysis.analyse(ctx, config, iteration)
+      val result = StaticAnalysis.analyse(ctx, config, iteration, analysisResult.lastOption)
```

Comment:
It's not safe to feed the previous iteration's results in like this - since the IR has been modified, the previous results will not be valid for many cases.

Reply:
At this point only IndirectCalls would be replaced by DirectCalls and the domain recalculated so any other node would have the result of the analysis. Why would it not be valid?

Reply:
Anything after a resolved call would not be valid

Reply:
The results fed are intraprocedural so they should not change. They will be undefined for new functions that have not been analysed but this case is handled

Reply:
The VSA depends on the ConstantPropagation results though, and the ConstantPropagation is interprocedural.